assets/scss/app.scss

```diff
@@ -1,4 +1,5 @@
 @import "$$theme";
+@import "node_modules/govuk-frontend/components/textarea/textarea";
```

Comment:
I think node_modules can be a little unreliable see here
https://www.npmjs.com/package/npm-sass
can you try this https://github.com/Shopify/slate/issues/582

Reply:
Thanks